### DISCUSSION TITLE
Glitchy and complex rhythms with Oi, Grandad!

### DISCUSSION THREAD

**Discussion by RickTaylor79** [publicsamples/Oi-Grandad#31] [Ideas]
On my Youtube channel a viewer asked:

_"I’m very interested in getting that granular Autechre feel to drum loops but struggling to get them sequenced right. Im assuming I’ll have to sample the sound and then cut it in my DAW."_

Since this reply is quite long & detailed, I'm posting it here in the hope that others will benefit from it too.

Generally speaking you'll have more control over things chopping in your DAW. Oi, Grandad!'s position control isn't that precise, especially when working with longer loops. However if you're willing to accept a few limitations then I'd say it can be used in this way. All of the public demos of the synth are straight recordings with no chopping, and while not of Autechre quality, glitchy drums and complex sequences are well represented.  

This isn't a guide as such, just a few suggestions and feature explanations that should point you in the right direction:

**Smoother loops**

The position, pitch and grain values are the keys to getting smooth(er) loop playback. Generally speaking a low grain size is preferable, especially when working with drums. Sometimes looping the entire sample from beginning to end can sound wrong, and it’s hard to get an accurate loop. In these cases, you can try setting the position modulation at 0.75. This will loop half the sample and in many cases works better

**Modulators can be triggered by other modulators**

Each of the modulators has a “Trig” menu that allows you determine how it is triggered (when it starts). They can be free running, triggered on midi note or by another modulator. This allows us to use Oi, Grandad! as kind of loop player when combined with the position control. For a brief example see this tutorial:

https://www.youtube.com/watch?v=2nC1YTvoH5o

**Modulating modulators**

One of my favourite Autechre tracks is [Fold4, Wrap5](https://www.youtube.com/watch?v=vUioVGqfu6s) with its shifting, pendulous rhythms. We can do a similar thing with OG by modulating the tempo of a modulator. 

**Pitch Sync**

When pitch sync is enabled OG will attempt to lock the pitch to your DAWs tempo. This can, in some cases, help with looping. When pitch is modulated, It can be use to create quantised “melodies” and various glitchy effects.

**Envelope follower**

Each voice has an envelope follower at the end of its signal chain. This can be used as be a modulation source (listed as G1, G2 etc in the mod menus). You can modulate parameters directly, or use it to trigger other modulators.

**Round Robin**

With the round robin switch enabled OG cycles between each of the 4 voices with each note triggered. Try loading some samples and playing it with an arpeggiator to get a feel for what this mode can do. 

**Crossfade**

When xfade is enabled a crossfader is applied to the volume of the 4 voices. We can use this to make smooth transitions between different sounds. The xfade position can be modulated, leading to interesting results. 

**Mod chains** 

[Oi, Grandad! v2](https://github.com/publicsamples/Oi-Grandad/releases/tag/2.1-b) features a “bank” of 8 presets for each modulator, further we can modulate between these presets. You can think of it as crossfading sequential switch. It allows for extremely complex & evolving modulations  A future update will include better visual feedback as currently it’s kind of hard to tell what’s going on.. 

Hopefully this helpful. If you want me to expand on anything, just ask and feel free to post any tricks for complex percussion you've discovered :)

**Comment by myvisualvoice** [comment]
Wow, Autechre Fan 🎹❣️
I love their COMPLEX out-of-this-world sounds.
My favourite one of their album, LP5, is "Corc".
But their "Clipper" from Tri Repetae (1995) leaves me wondering how the hell they create their SOUNDS !

**Comment by RickTaylor79** [comment]
Yeah, they're a huge influence for me, especially when it comes to instrument design. 

Tri Repetae is a bone fide classic, you have great taste!

**Comment by myvisualvoice** [comment]
Thank you.
Nice to know you.

________________________________
